Makefile

```diff
@@ -25,7 +25,7 @@ GIT_DESCRIBE ?= $(shell git describe --tags --abbrev=8 --always --long --dirty 2
 VERSION_TAG ?= $(shell echo "$(GIT_DESCRIBE)" | cut -f1 -d-)
 APPVERSION_M=2
 APPVERSION_N=4
-APPVERSION_P=6
+APPVERSION_P=7
```

Comment:
Why increase the version here?

Reply:
For testing. Need to provide DAL testing to other teams so need to have separate version for clarity. So we can say version xxx supports DAL attestation.